### PR TITLE
Title page

### PR DESCRIPTION
**Pull request by zainafzal08**
This adds some styles for the main menu, settings and loading pages. It also locks the entire game to a 16:9 aspect ratio. This means that we don't have to worry about responsive design but by tying styles to window width we can still make the most of the real estate we have.

### CODE REVIEW

File: src/main-menu.ts [hunk 1]
```diff
@@ -1,24 +1,98 @@
...
       display: flex;
       align-items: center;
       justify-content: center;
       flex-direction: column;
+      background: rgb(240, 200, 98);
+    }
+    h1 {
+      font-family: "Poppins", sans-serif;
+      font-size: 12vw;
+      position: absolute;
```
Comment (josephkimsungha) [inline src/main-menu.ts:21]: Lol feels weird to dictate the position of all H1's but ok

File: src/main-menu.ts [hunk 2]
```diff
@@ -1,24 +1,98 @@
...
+      padding: 0 15vw 15vw 15vw;
+      transform: translate(-50%);
+    }
+    #settings-button {
+      right: 8%;
+      top: 35%;
+      padding-bottom: 25vw;
+    }
+    :host([play-hovered]) #play-button,
+    #play-button:hover {
```
Comment (josephkimsungha) [inline src/main-menu.ts]: nit: we could merge all the hover rules into one group e.g.
```
    #start-button:hover,
    #back-button:hover,
    :host([play-hovered]) #play-button,
    #play-button:hover,
    :host([settings-hovered]) #settings-button,
    #settings-button:hover {
      opacity: 100%;
    }
```

File: src/main-menu.ts [hunk 3]
```diff
@@ -1,24 +1,98 @@
...
+    }
+    :host([settings-hovered]) #settings-button,
+    #settings-button:hover {
+      opacity: 100%;
+    }
+    .bg {
+      width: 100%;
+      height: 100%;
+      position: absolute;
+      top: 0;
```
Comment (josephkimsungha) [inline src/main-menu.ts:61]: Does this work if we're in a different aspect ratio? Or does absolute positioning position relative to the nearest relative positioned parent... I forget

File: src/main-menu.ts (hunk 2 above)
Comment (josephkimsungha) [reply src/main-menu.ts]: At this stage, I almost wonder if we should be using a class lol

File: src/main-menu.ts [hunk 4]
```diff
@@ -1,24 +1,98 @@
-import { LitElement, PropertyValueMap, css, html } from "lit";
-import { customElement, state } from "lit/decorators.js";
+import { LitElement, css, html } from "lit";
+
```
Comment (josephkimsungha) [inline src/main-menu.ts]: nit: remove

File: src/main-menu.ts [hunk 5]
```diff
@@ -1,24 +1,98 @@
...
+    }
+    :host([settings-hovered]) #mirror-sheen {
+      transform: translate(30px, -40px);
     }
   `;
 
   @state()
   private settingsOpen = false;
-
+  @property({ type: Boolean, reflect: true, attribute: "play-hovered" })
```
Comment (josephkimsungha) [inline src/main-menu.ts:83]: optional: should we have properties above state?

**Review by josephkimsungha — APPROVED** [review]

File: src/main-menu.ts (hunk 1 above)
Comment (zainafzal08) [reply src/main-menu.ts:21]: Updated to use a class

File: src/main-menu.ts (hunk 2 above)
Comment (zainafzal08) [reply src/main-menu.ts]: Yeah look in retrospect it's so wild i didn't just use a class lol. updated.

File: src/main-menu.ts (hunk 3 above)
Comment (zainafzal08) [reply src/main-menu.ts:61]: Resizing the window width seems to work, if you reduce the height past 900px tho it just overflows the window which i feel like is ok.

**Pull request merged**